src/dodal/devices/backlight.py

```diff
@@ -18,3 +17,21 @@ def set(self, position: int) -> StatusBase:
         else:
             status &= self.toggle.set("On")
         return status
+
+
+class VmxmBacklight(Device):
+    """Simple device for VMXm to control the backlight."""
+
+    OUT = 0
+    IN = 1
+
+    pos: EpicsSignal = Component(EpicsSignal, "-DI-IOC-02:LED:INOUT")
+    toggle: EpicsSignal = Component(EpicsSignal, "-EA-OAV-01:FZOOM:TOGGLE")
+
+    def set(self, position: int) -> StatusBase:
+        status = self.pos.set(position)
+        if position == self.OUT:
+            status &= self.toggle.set("Off")
+        else:
+            status &= self.toggle.set("On")
+        return status
```

Comment:
This device is the same as I03's backlight other than the PV names. I think we should make a base Backlight device where the PV names are parameters

Reply:
I agree, but in Ophyd V1 it's actually surprisingly tricky to do that (due to some of the `Component` magic).

Suggest spinning that out into an issue if there isn't already one?